update_pipeline_details.php

```diff
@@ -242,11 +242,41 @@ function sort_datestamp($a, $b) {
             $config['twitter_access_token'],
             $config['twitter_access_token_secret'],
         );
+        // loop through pages of twitter api results
+        $page = 0;
+        $max_pages = 100;
 
-        // Post the tweets
+        $already_tweeted = false;
         foreach ($tweets as $tweet) {
-            $post_tweets = $connection->post('statuses/update', ['status' => $tweet]);
-            echo "Sent tweet: $tweet\n";
+            while ($page < $max_pages) {
+                $page++;
+                $params = [
+                    'count' => 200,
+                    'page' => $page,
+                    'exclude_replies' => true,
+                    'exclude_retweets' => true,
+                ];
+                $old_tweets = $connection->get('statuses/user_timeline', $params);
+                // Check for substring of the new tweet in old tweets
+                foreach ($old_tweets as $old_tweet) {
+                    preg_match('/.*?\(/', $tweet, $match);
```

Comment:
This regex looks odd at a glance. Is it right?

Reply:
I made it stricter now (I think/hope).

Reply:
Looks good! I hadn't properly understood what it was doing but I get it now. I thought that you were using the regex to check for the name + version directly, had my noticed the second strpos check.